ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -26,7 +26,7 @@ public void WhenInitialized_FindCellStateOfCell(Cell cell)
             var universe = new Universe(3, 3);
             var expected = CellState.Dead;
 
-            var result = universe.GetCellAtLocation(cell.Location.X, cell.Location.Y).CellState;
+            var result = universe.GetCellAtLocation(cell.Location).CellState;

```

Comment:
try not to "reach in" too far i.e. too many `.` in one line

```suggestion
            var cell = universe.GetCellAtLocation(cell.Location);
            var result = cell.CellState;
```